httpie/output/formatters/json.py

```diff
@@ -17,6 +17,10 @@ def format_body(self, body, mime):
         ]
         if (self.kwargs['explicit_json'] or
                 any(token in mime for token in maybe_json)):
+            if 'xssi_prefix' in self.kwargs:
```

Comment:
This will always be true since you defaulted it in the cli arguments. I think it'd be better to reverse this logic a bit:

```python
xssi_prefix = self.kwargs["xssi_prefix"]

if xssi_prefix and body.startswith(xssi_prefix):
  body = body[len(xssi_prefix):]
```